scripts/quests/sandoria/Spice_Gals.lua

```diff
@@ -118,9 +119,13 @@ quest.sections =
             onEventFinish =
             {
                 [725] = function(player, csid, option, npc)
-                    player:delKeyItem(xi.ki.RIVERNEWORT)
-                    npcUtil.giveItem(player, xi.items.MIRATETES_MEMOIRS)
-                    quest:setVar(player, 'Option', getConquestTally())
+                    if player:getFreeSlotsCount() > 0 then
+                        player:delKeyItem(xi.ki.RIVERNEWORT)
+                        npcUtil.giveItem(player, xi.items.MIRATETES_MEMOIRS)
+                        quest:setVar(player, 'Option', getConquestTally())
+                    else
+                        player:messageSpecial(southSandyID.text.ITEM_CANNOT_BE_OBTAINED, xi.items.MIRATETES_MEMOIRS)
```

Comment:
I am pretty sure this can be removed since npcUtil should handle the cannot be obtained message if full inv..?

Reply:
> I am pretty sure this can be removed since npcUtil should handle the cannot be obtained message if full inv..?

Ah yeah I updated the code, can check now :)